bioblend/galaxy/invocations/__init__.py

```diff
@@ -181,6 +212,32 @@ def get_invocation_report(self, invocation_id):
         url = self._make_url(invocation_id) + '/report'
         return self._get(url=url)
 
+    def get_invocation_report_pdf(self, invocation_id):
```

Comment:
Can you combine this with the method above by including a `format`argument which defaults to markdown but can also be set to pdf?

The PDF also only renders if the weasyprint dependency is installed, see: https://github.com/galaxyproject/galaxy/pull/9651. Can you raise an error if this is the case?